content/en/monitors/types/error_tracking.md

```diff
@@ -37,76 +37,106 @@ To create an Error Tracking monitor in Datadog, navigate to [**Monitors** > **Ne
 
 There are two types of alerting conditions you can configure your Error Tracking monitor with:
 
-| Alerting&nbsp;condition     | Description    | 
+| Alerting&nbsp;condition     | Description    |
 | ---  | ----------- |
-|High Impact| Alert on issues with a high number of impacted end users. For example, alert for your service whenever more than 500 users are impacted by this error. |
 |New Issue| Alert when an issue occurs for the first time or a regression occurs. For example, alert for your service whenever more than 2 users are impacted by a new error. |
+|High Impact| Alert on issues with a high number of impacted end users. For example, alert for your service whenever more than 500 users are impacted by this error. |
 
-### Define the search query
+### Define alert conditions
 
 {{< tabs >}}
-{{% tab "High Impact" %}}
-High Impact monitors alert on issues that are **For Review** or **Reviewed** and that meet your alerting conditions. Read more about [Issue States][1].
 
-1. Build a search query using the same logic as the [Error Tracking Explorer search][2] for the issues' error occurrences.
-2. Choose the metric you want to monitor. There are three suggested filter options to access the most frequently used facets:
+{{% tab "New Issue" %}}
+#### Issues to alert on
+
+New issue monitors alert on issues that are **For Review** and that meet your alerting conditions. Read more about [Issue States here][1]. As regressions are transitioned to **For Review** automatically, they are automatically monitored with New Issue monitors.
+
+Select **All**, **Browser**, **Mobile**, or **Backend** issues and construct a search query using the same logic as the [Error Tracking Explorer search][2] for the issues' error occurrences.
+
+<div class="alert alert-info"><strong>Note</strong>: New Issue monitors only consider issues that were created or regressed after the monitor was created or last edited, with a 24-hour lookback period.</div>
+
+#### Define alert threshold
+
+Choose one of the following options:
+
+**Alert on all new issues**:
```

Comment:
This is optional, but from the Staging preview, it's difficult to distinguish the header and the options. Putting the two options inside a collapse content makes it a little clearer on what actions the reader can take.

```suggestion
{{% collapse-content title="Alert on all new issues" level="p" %}}

```